emf/src/main/kotlin/com/strumenta/kolasu/emf/MetamodelBuilder.kt

```diff
@@ -229,7 +232,8 @@ class MetamodelBuilder(packageName: String, nsURI: String, nsPrefix: String, res
     private fun registerReference(
```

Comment:
Maybe this name is confusing. Perhaps we could call it `Link`, where `Link` is the generic thing that could either be a Containment or a Reference

Reply:
Yes, makes sense.